code/RedirectedURL.php

```diff
@@ -35,7 +35,21 @@ class RedirectedURL extends DataObject implements PermissionProvider {
 		'To',
 	);
 
-	public function getCMSFields() {
+	public function validate()
+    {
```

Comment:
please use tabs to indent as the current file does.

Reply:
Done